src/app/zap-templates/zcl/fabric-cluster.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2021 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+  <cluster>
+    <domain>General</domain>
+    <name>Fabric</name>
+    <code>0xfc03</code> // Change to correct clusterID once it is decided in the spec #2396
+    <define>FABRIC_CLUSTER</define>
+    <description>The Fabric Cluster defines a cluster that allows you to fetch and manipulate the list of paired fabrics.</description>
+
+    <attribute side="server" code="0x0000" define="FABRICS_LIST" type="ARRAY" entryType="FabricDescriptor" length="254" writable="false" optional="false">fabrics list</attribute>
+
+    <command source="client" code="0x00" name="UpdateFabricLabel" optional="false">
+      <description>Update Label on fabric</description>
+      <arg name="fabricLabel" type="OCTET_STRING" length="32"/> //TODO: change this to CHAR_STRING once it is supported #6112
+    </command>
+
+    <command source="client" code="0x01" name="RemoveFabric" optional="false">
+      <description>Remove paired fabric</description>
+      <arg name="fabricId" type="FABRIC_ID"/>
+      <arg name="nodeId" type="NODE_ID"/>
+      <arg name="vendorId" type="INT16U"/>
+    </command>
+
+    <command source="client" code="0x02" name="GetFabricId" optional="false">
+      <description>Sets the fabric ID of that admin's fabric. Should be called post pairing and can only be called once</description>
```

Comment:
Sets -> gets ?

How do you set it?

Reply:
Because we have not implemented generating the fabricId... Temporarily will be using the randomly generated nodeID that the commissioner generates.

Reply:
So when pairing is complete and we store the destination node ID (accessory node ID) and adminID in the pairing table. I will store the source node ID (commissioner node ID) and the vendor ID.

Reply:
So the node id is also the fabric id ?

Reply:
I get it now, there are both src and dest node ids and src is used as fabric id. Thanks.

Reply:
To explain, because we have not implemented addOptCert + fabric index yet and I wanted to add initial support for fabric management, here is what I did (separate patch):
- when commissioner pairs with device, on rendez vous code when we create a new admin and add an entry into the admin pairing table along with the device node Id, I add the commissioner node id + vendor for that fabric entry. So that is how I set a fabricID for now (that along with vendor ID + device node ID will be unique to differentiate between fabrics for now). 
- when we call updateFabricLabel, on incoming message I use those same fields (instead of fabric index) to figure out what fabric is the correct one and update its label.
- for removeFabric, since fabric index is not implemented yet, I have the commissioner pass the three fields of the fabric they want removed (fabric ID, node ID, vid). I know this is not up to spec but without fabric index, needed other info.
- GetFabricID is not in the spec, I only need it for now so that commissioner can know which fabric is their own when they read the fabrics list attribute. Once again, once fabric index + addOptCert is implemented, this would go away.

I tried my best to follow the spec but without other parts implemented, this was easiest path to getting basic functionality without adding a lot of throwaway code (use fields we already have access to during pairing: vid, device node id and commissioner node id to add an entry to fabrics list).